src/core/jvm/insn/LookupSwitchInstruction.js

```diff
@@ -0,0 +1,29 @@
+const PaddedInstruction = require('./PaddedInstruction');
+const OffsetPair = require('../OffsetPair')
+
+export default class LookupSwitchInstruction extends PaddedInstruction {
+
+  constructor(idx, opcode) {
+    super(idx, opcode);
+    this.offsetPairs = [];
+    this.defaultOffset = 0;
+  }
+
+  get size() {
+    return (super.size + 8) + (8 * this.offsetPairs.length);
+  }
+
+  read(buffer) {
+    super.read(buffer);
+    while (offsetPairs.length > 0) {
+        offsetPairs.pop();
```

Comment:
`offsetPairs` is undefined in this scope. you need to reference `this.offsetPairs`